benchmarks/src/main.rs

```diff
@@ -61,8 +49,8 @@ fn get_nth_percentile(durations: &[Duration], n: f64) -> Duration {
     durations[std::cmp::max(index, 1) - 1]
 }
 
-fn score_test(durations: Vec<Duration>, name: &str, task_size: f64, more_info: bool) -> f64 {
-    let median = get_nth_percentile(&durations, (*MAX_ITERATIONS / 2) as f64);
+fn score_test(mut durations: Vec<Duration>, name: &str, task_size: f64, more_info: bool) -> f64 {
+    let median = get_nth_percentile(&durations, durations.len() / 2 as f64);
```

Comment:
given that this function accepts a percentile, don't actually need to use len at all and can just say `50.0`.

Reply:
Ah, true! I think it might actually be incorrect like this when the length is not 100